opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/MailTransportMonitor.adoc

```diff
@@ -0,0 +1,182 @@
+| `start-tls`            | Use the STARTTLS command (if supported or required by the server) to switch the connection 
+                            to a TLS-protected connection before issuing any login commands                             | optional | `"false`
+|===
+
+.Options for sendmail-message
+[options="header, autowidth"]
+|===
+| Attribute | Description                                                                                               | Required | Default value
+| `to`      | The destination address                                                                                   | optional | `root@localhost`
+| `from`    | The address to insert into the From: field                                                                | optional | `root@[127.0.0.1]`
+| `subject` | The message subject                                                                                       | optional | `OpenNMS Test Message`
+| `body`    | The body of the message                                                                                   | optional | `This is an OpenNMS test message.`
+|===
+
+.Options for readmail-test
+[options="header, autowidth"]
+|===
+| Attribute          | Description                                                                                      | Required | Default value
+| `debug`            | Show additional debug output                                                                     | optional | `true`
+| `mail-folder`      | The folder or IMAP label to check for mail                                                       | optional | `INBOX`
+| `subject-match`    | Substring match in email subjects when looking for a specific email                              | optional | `no default`
+| `attempt-interval` | Attempt to read email after this many milliseconds have passed, also used for retry interval     | optional | `1000`
+| `delete-all-mail`  | Delete all read mail after a successful match                                                    | optional | `false`
+|===
+
+.Options for readmail-host
+[options="header, autowidth"]
+|===
+| Attribute   | Description                                                                                             | Required | Default value
+| `host`      | The target host for reading mail                                                                        | optional | `${ipaddr}`
+| `port`      | the appropriate port for the protocol                                                                   | optional | `110`
+|===
+
+.Options for readmail-protocol
+[options="header, autowidth"]
+|===
+| Attribute    | Description                                                                                             | Required | Default value
+| `transport`  | The transport ptotocol to use.  One of:  pop3 pop3s imap imaps                                          | optional | `pop3`
```

Comment:
```suggestion
| `transport`  | The transport ptotocol to use. One of: pop3 pop3s imap imaps                                          | optional | `pop3`
```
Are these literals? If not, please capitalize: POP3 POP3S IMAP IMAPS

Reply:
Yes, these are literal.